src/app/app.component.ts

```diff
@@ -0,0 +1,29 @@
+import { TranslateService } from '@ngx-translate/core';
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { LoaderService } from './core/services/loader.service';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+export class AppComponent {
+  title = 'pms-application';
+  isLoading = false;
+
+  constructor(
+    public loaderService: LoaderService,
+    private cdref: ChangeDetectorRef,
+    public translate: TranslateService
+  ) {}
+
+  ngOnInit() {
+    this.title = 'pms-application';
+    this.loaderService.isLoading.subscribe((isLoad) => {
```

Comment:
Every time to visit this page will be created a new subscription. This leads to performance issue. So you should pay attention to closing them. It might be unsubscribing ( in the onDestroy method) or approaches from the article https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f. In the projects' life, we use take* more often than others.